src/huggingface_hub/keras_mixin.py

```diff
@@ -27,6 +31,10 @@ def save_pretrained_keras(
         Specify directory in which you want to save the Keras model.
     config (:obj:`dict`, `optional`):
         Configuration object to be saved alongside the model weights.
+    include_optimizer(:obj:`bool`, `optional`):
+        Whether or not to include optimizer in serialization.
+    model_save_kwargs(:obj:`dict`, `optional`):
+        Arguments other than default arguments that can be passed to tf.keras.models.save_model().
```

Comment:
For consistency with other mixins and methods, maybe let's use 

`model_save_kwargs will be passed to `tf.keras.models.save_model()``

(e.g. https://github.com/huggingface/huggingface_hub/blob/main/src/huggingface_hub/hub_mixin.py#L49)